Validate contact form

abdulrahmanshr75 opened the pull request:
implemented a simple validation : 
- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error - and the form is not sent.

abdulrahmanshr75 committed:
Implementing the Email error message

SirriRyisa committed:
Fixed Css

abdulrahmanshr75 committed:
Fixed linters and style

HENRYKC24 commented on style.css:
```diff
@@ -19,6 +19,17 @@ header {
+  padding: 12px 8px;
+  font-size: 14px;
+  border-radius: 8px;
+  background-color: #ff0905ca;
```
- [ ] Great job implementing this. Your method is top-notch for the error message. I want you to even do more by making sure that the background color of your error message blends with the design of your website. I think the current color is off. Using a stray color normally causes color conflict which is not a good UI approach I think. This color would have been better if you had a touch of red in your current design of the website. What do you think?
![image](https://user-images.githubusercontent.com/45668340/147144937-6512ce5d-677f-4acd-aa4e-366afb5e5aaa.png)

HENRYKC24 reviewed (changes requested):
# STATUS: CHANGES REQUESTED:recycle:
![Close](https://media.giphy.com/media/nYYtQdLqwwlTe3ccKh/giphy.gif)
## Hi @abdulrahmanshr75 :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Change color to match the figma design

i changed the red color to the red that is in the figma design suggestions to blend the colors

HENRYKC24 reviewed (approved):
# STATUS: APPROVED:heavy_check_mark:
![Cool](https://media.giphy.com/media/lrDMG46JjKiovLbkn8/giphy.gif)
## Hi @abdulrahmanshr75 :wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
